paper-collapse-item.html

```diff
@@ -133,15 +134,18 @@
 			}
 		},
 
+		
+		_opendChanged: function(){
+			this.fire('toggle', this);
+		},
 		// Private methods
 		/**
 		 * Fired whenever the status is changed (opened/closed)
 		 *
 		 * @event toggle
 		 */
 		_toggleOpened: function(e) {
-			this.opened = !this.opened;
-			this.fire('toggle', this);
+			this.set('opened',!this.opened);
```

Comment:
Could please add a space after the comma (for code consistency)? Thanks

Reply:
Sure , I will do it later this day :)